examples/ini.js

```diff
@@ -0,0 +1,47 @@
+const fs = require("fs");
+const ini = require("ini");
+const utils = require('../utils');
+const path = require('path');
+const Store = require('../');
+
+// Will be called in context of store
+function writeFile() {
+  console.log("writeFile");
```

Comment:
Please either remove this line or use single quotes:

```suggestion
  console.log('writeFile');
```